reference/session/functions/session-module-name.xml

```diff
@@ -39,12 +46,40 @@
  <refsect1 role="returnvalues">
   &reftitle.returnvalues;
   <para>
-   Zwraca nazwę obecnego modułu sesji.
+   Zwraca nazwę obecnego modułu sesji, &return.falseforfailure;.
   </para>
  </refsect1>
 
-</refentry>
+ <refsect1 role="changelog">
+  &reftitle.changelog;
+  <informaltable>
+   <tgroup cols="2">
+    <thead>
+     <row>
+      <entry>&Version;</entry>
+      <entry>&Description;</entry>
+     </row>
+    </thead>
+    <tbody>
+     <row>
+      <entry>8.0.0</entry>
+      <entry>
+       Parametr <parameter>module</parameter> przyjmuję teraz wartość null.
```

Comment:
"przyjmuję" -> "przyjmuje", choć ja napisałbym "akceptuje" lub "pozwala teraz na wartość null". Chodzi o to, żeby nie pozostawić wątpliwości że null jest dozwolony jako dodatkowa wartość, a nie jedyna.